documentation/docs/03-routing.md

```diff
@@ -108,6 +111,8 @@ export async function load({ params }) {
 
 During client-side navigation, SvelteKit will load this data from the server, which means that the returned value must be serializable using [devalue](https://github.com/rich-harris/devalue).
 
+Like `+page.js`, `+page.server.js` can export [page options](/docs/page-options) — `prerender`, `hydrate`, `router` and `ssr`.
```

Comment:
I did the same mistake 😄 
```suggestion
Like `+page.js`, `+page.server.js` can export [page options](/docs/12-page-options) — `prerender`, `hydrate`, `router` and `ssr`.
```

Reply:
nope, the numeric prefix is incorrect! reverted

Reply:
yeah just realized, sry for the noise